meilisearch/Cargo.toml

```diff
@@ -3,7 +3,7 @@ authors = ["Quentin de Quelen <quentin@dequelen.me>", "Clément Renault <clement
 description = "Meilisearch HTTP server"
 edition = "2021"
 license = "MIT"
-name = "meilisearch-http"
+name = "meilisearch"
 version = "0.30.0"
 
 [[bin]]
```

Comment:
Could you please remove this `[[bin]]` section, please? 😉

Reply:
@Kerollmops 
`[[bin]]
name = "meilisearch"
path = "src/main.rs"`

This entire section needs to be removed?

Reply:
Yes! You should be able to remove everything because, by default, cargo is going to name the binary after the name of the crate (meilisearch here).